src/Symfony/Component/Validator/Constraints/UniqueValidator.php

```diff
@@ -21,6 +26,13 @@
  */
 class UniqueValidator extends ConstraintValidator
 {
+    private ?PropertyAccessorInterface $propertyAccessor;
+
+    public function __construct(PropertyAccessorInterface $propertyAccessor = null)
+    {
+        $this->propertyAccessor = $propertyAccessor;
+    }
```

Comment:
As the minimal requirement for 6.3 is PHP 8.1, I guess this could be using constructor property promotion 👌